pkg/server/options/options.go

```diff
@@ -329,7 +329,18 @@ func (o *Options) Complete() (*CompletedOptions, error) {
 		o.Extra.BatteriesIncluded = bats.List()
 	}
 
-	o.Cache.Complete(completedGenericControlPlane.SecureServing)
+	cacheServerEtcdOptions := *completedGenericControlPlane.Etcd
+	o.Cache.Server.Etcd = &cacheServerEtcdOptions
```

Comment:
nit: technically a completion func should do something like:
```
if o.Cache.Server.Etcd == nil { 
  o.Cache.Server.Etcd = &cacheServerEtcdOptions
}
```
Think about future customizers.

Reply:
mhm, `o.Cache.Server.Etcd ` cannot be nil, it gets populated in `Server: cacheoptions.NewOptions(rootDir),`